test-packages/platform-feature-tests/cypress.config.js

```diff
@@ -1,13 +1,28 @@
 const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
+  pageLoadTimeout: 300000,
+  reporter: 'cypress-mochawesome-reporter',
+  video: false,
+  reporterOptions: {
+    reportDir:'cypress/reports',
+    charts: true,
+    overwrite: false,
+    html: true,
+    json: false,
+    timestamp: "mmddyyyy_HHMMss",
+    reportPageTitle: 'Cypress Inline Reporter',
+    embeddedScreenshots: true, 
+    inlineAssets: true, //Adds the asserts inline
+  },
+
 	e2e: {
 		async setupNodeEvents(on, config) {
-
+      require('cypress-mochawesome-reporter/plugin')(on);
 		},
 		appeals_beta_base_url: 'https://appeals-service-test.planninginspectorate.gov.uk', //TODO: make this an env var?
```

Comment:
could we pull the url from an env var, so we point tests to url from pipeline 

also think horizon_base_url is gone now so can be removed?

Reply:
.env file created and added base url.
horizon_base_url removed from config